src/discordtipbot/discordbot.cr

```diff
@@ -152,6 +152,14 @@ class DiscordBot
         end
       end
     end
+
+    # Make use of the status to display info
+    spawn do
+      sleep 2
+      Discord.every(10.seconds) do
```

Comment:
The rate limit for presence changes is 12 seconds. See just below here: https://discordapp.com/developers/docs/topics/gateway#activity-object-example-activity-with-rich-presence

Reply:
This data is probably pretty unlikely to show meaningful change at that rate anyways. I would back this off even further to `1.minute`. No reason to fill the RL bucket that quickly, when there's other things we may need to be sending over the websocket as well (heartbeats, etc)

Reply:
Oops. I thought I had changed it to once per minute before committing